### PR TITLE
chore(kit): add xxs size for progress bar to match xxl

### PR DESCRIPTION
**Pull request by evantrimboli**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

N/A

## What is the new behavior?
Add a new size.

### CODE REVIEW

**Comment by evantrimboli** [comment]
In my project I was using the progress bar with `height: 0.125rem` as a small indeterminate loader.

File: projects/kit/components/progress/progress-bar/progress-bar.component.less [hunk 1]
```diff
@@ -82,6 +83,10 @@
             });
         }
 
+        &[data-size='xxs'] {
+            height: 0.125rem;
```
Comment (nsbarsukov) [inline projects/kit/components/progress/progress-bar/progress-bar.component.less:87]: Unfortunately, this size is not compatible with our design system 😢 
I think we should not merge this PR 🤔

Comment (nsbarsukov) [reply projects/kit/components/progress/progress-bar/progress-bar.component.less:87]: @evantrimboli 
But it not big problem. You can easily set any size you wish for `ProgressBar`.

```html
<progress
    tuiProgressBar
    max="100"
    [value]="50"
    class="progress"
></progress>

```

```css
.progress {
    height: 0.125rem;
}
```

Comment (evantrimboli) [reply projects/kit/components/progress/progress-bar/progress-bar.component.less:87]: Ok no worries. I'll close this out.

**Pull request closed without merging**